client/landing/stepper/declarative-flow/internals/steps-repository/choose-a-domain/style.scss

```
@@ -336,9 +336,24 @@ $videopress-background: #010101;
 
 	.register-domain-step__next-page-button {
 		padding-bottom: 8px !important;
+		margin: 20px 0;
 	}
 
 	.is-borderless {
 		border: none;
 	}
+
+	.domain-suggestion,
+	.domain-suggestion:hover,
+	.register-domain-step__next-page {
```

Just a detail, adding the `.featured-domain-suggestion` looks better while loading
```suggestion
	.featured-domain-suggestion,
	.featured-domain-suggestion:hover,
	.register-domain-step__next-page {
```

---

Looks good! Thanks!